components/brave_wallet_ui/components/desktop/views/accounts/account.tsx

```diff
@@ -94,7 +95,10 @@ export const Account = (props: Props) => {
 
   // custom hooks
   const getBalance = useBalance(networkList)
-  const { copied, copyText } = useCopy()
+  const {
+    isCopied: copied,
+    temporaryCopyToClipboard: copyText
+  } = useTemporaryCopyToClipboard(1500)
```

Comment:
Is `1 1/2` seconds long enough for someone to copy and paste their wallet address somewhere?
Curious why we need this for addresses?

Reply:
1.5 seconds was the time set by @muliswilliam 's other clipboard hook. 

@muliswilliam, you ok with me increasing the time for this?

Reply:
The difference here though is @muliswilliam `1.5` seconds was for displaying `Copied` tooltip.

This now uses that time to clear the `clipboard` https://github.com/brave/brave-core/pull/13146/files#diff-1524a199b29529703bbec790905dc4ce35f2f8842dd5aa033f4b08baac1a95aaR34

Maybe it should be separated or have an optional timeout prop to clear the clipboard if provided.

Reply:
Fixed in: [61327a9](https://github.com/brave/brave-core/pull/13146/commits/61327a9c2e6b3c9f3483087155c9263a431817f3)